en-US/snippets/2017/thanks_sep2017.lang

```diff
@@ -0,0 +1,17 @@
+## NOTE: Reference bug 1394945. Deadline: 15 Sept, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# The text will be stylized.
```

Comment:
Should we expose directly "THANK YOU"?

I think the comment is a bit misleading, I would just add that it's used as a title.

Reply:
OK, let me do that. Chinese will look really weird, :).